.gitignore

```diff
@@ -20,6 +20,7 @@ policy.wasm
 .go
 release-notes.md
 bundle.tar.gz
+/docs/website/.hugo_build.lock
```

Comment:
[nit] Looks unrelated, can we drop this? 😉

Reply:
Sure. This one had been bugging me, as it always shows up when I build the docs locally to check that my changes look ok 😅 .